src/source/Rtcp/RtcpPacket.c

```diff
@@ -72,3 +75,63 @@ STATUS rtcpNackListGet(PBYTE pPayload, UINT32 pPayloadLen, PUINT32 pSenderSsrc,
     LEAVES();
     return retStatus;
 }
+
+STATUS rtcpNackListGetSsrcs(PBYTE pPayload, UINT32 payloadLen, PUINT32 pSenderSsrc, PUINT32 pReceiverSsrc) {
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    INT32 i = RTCP_NACK_LIST_LEN, j;
+    UINT16 currentSequenceNumber;
+    UINT16 sequenceNumberCount = 0, BLP;
+
+
+    CHK(pPayload != NULL && pSenderSsrc != NULL && pReceiverSsrc != NULL, STATUS_NULL_ARG);
+    CHK(payloadLen >= RTCP_NACK_LIST_LEN && (payloadLen % 4 == 0), STATUS_RTCP_INPUT_NACK_LIST_INVALID);
+
+    *pSenderSsrc = getInt32(*(PUINT32) pPayload);
+    *pReceiverSsrc = getInt32(*(PUINT32) (pPayload + 4));
+
+CleanUp:
+    LEAVES();
+    return retStatus;
+}
+
+STATUS rtcpNackListGetSeqNums(PBYTE pPayload, UINT32 payloadLen, PUINT16 pSequenceNumberList, PUINT32 pSequenceNumberListLen) {
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    INT32 i = RTCP_NACK_LIST_LEN, j;
+    UINT16 currentSequenceNumber;
+    UINT16 sequenceNumberCount = 0, BLP;
```

Comment:
I would prefer not using all caps params - why not lowercase it? Same for the other functions